include/language.php

```diff
@@ -460,21 +729,18 @@ public function set_home_url( $search_url, $home_url ) {
 	 * @return void
 	 */
 	public function set_url_scheme() {
-		if ( ! empty( $this->home_url ) ) {
-			$this->home_url = set_url_scheme( $this->home_url );
-		}
+		$props = array( 'home_url', 'search_url', 'flag_url', 'custom_flag_url' );
 
-		if ( ! empty( $this->search_url ) ) {
-			$this->search_url = set_url_scheme( $this->search_url );
-		}
+		foreach ( $props as $prop ) {
+			if ( empty( $this->$prop ) ) {
+				continue;
+			}
 
-		// Set url scheme, also for the flags.
-		if ( ! empty( $this->flag_url ) ) {
-			$this->flag_url = set_url_scheme( $this->flag_url );
-		}
+			$url = set_url_scheme( $this->$prop );
 
-		if ( ! empty( $this->custom_flag_url ) ) {
-			$this->custom_flag_url = set_url_scheme( $this->custom_flag_url );
+			if ( ! empty( $url ) ) {
+				$this->$prop = $url;
+			}
```

Comment:
Because `set_url_scheme()` returns `string` (not `non-empty-string`), and PHPStan doesn't like it because those properties are tagged with `non-empty-string`. The other solution is to do this:
```php
$url = set_url_scheme( $this->$prop );
/** @var non-empty-string $url */
$this->$prop = $url;
```